subsys/bluetooth/host/classic/shell/bredr.c

```diff
@@ -248,52 +329,321 @@ static const struct bt_l2cap_chan_ops l2cap_ops = {
 	.disconnected	= l2cap_disconnected,
 };
 
-static struct bt_l2cap_br_chan l2cap_chan = {
-	.chan.ops	= &l2cap_ops,
-	 /* Set for now min. MTU */
-	.rx.mtu		= 48,
+#define BT_L2CAP_BR_SERVER_OPT_RET           BIT(0)
+#define BT_L2CAP_BR_SERVER_OPT_FC            BIT(1)
+#define BT_L2CAP_BR_SERVER_OPT_ERET          BIT(2)
+#define BT_L2CAP_BR_SERVER_OPT_STREAM        BIT(3)
+#define BT_L2CAP_BR_SERVER_OPT_MODE_OPTIONAL BIT(4)
+#define BT_L2CAP_BR_SERVER_OPT_EXT_WIN_SIZE  BIT(5)
+#define BT_L2CAP_BR_SERVER_OPT_HOLD_CREDIT   BIT(6)
+
+static int l2cap_accept(struct bt_conn *conn, struct bt_l2cap_server *server,
+			struct bt_l2cap_chan **chan);
+
+static struct bt_l2cap_br_server l2cap_server = {
+	.server = {.accept = l2cap_accept,},
+};
```

Comment:
Just format this the same way you've done with `l2cap_chan` below, i.e. put the `.accept = l2cap_accept,` on its own line.

Reply:
Updated.